File: Versao-Python/Desafio.py
def calcular_imposto(valor_salario):
    if (valor_salario >= 0 and valor_salario <= 1100):
        imposto = 0.05
    elif (valor_salario > 1100 and valor_salario <= 2500):
        imposto = 0.10
    else:
        imposto = 0.15
    return imposto * valor_salario

valor_salario = float(input())
valor_beneficios = float(input())

valor_imposto = calcular_imposto(valor_salario)

saida = valor_salario - valor_imposto + valor_beneficios
print(f"{saida:.2f}")

File: Versao-C#/Desafio.cs
using System;

public class Desafio {
    public static void Main() {
        float valorSalario = float.Parse(Console.Read())
        float valorBeneficios = float.Parse(Console.Read())

        float valorImposto = 0;
        if (valorSalario >= 0 && valorSalario <= 1100) {
            valorImposto = 0.05F * valorSalario;
        }

        else if (valorSalario > 1100 && valorSalario <= 2500 ) {
            valorImposto = 0.10F * valorSalario;
        }

        else {
            valorImposto = 0.15F * valorSalario;
        }

        float saida = valorSalario - valorImposto + valorBeneficio;
        Console.WriteLine(saida.ToString("0.00"));
    }
}